lua/lspconfig/julials.lua

```diff
@@ -38,9 +46,6 @@ local cmd = {
 configs.julials = {
   default_config = {
     cmd = cmd,
-    on_new_config = function(new_config, root_dir)
-      new_config.cmd_cwd = root_dir
```

Comment:
When is this necessary? I couldn't find a case where it was necessary.

Reply:
I added this. 

Try launching neovim in home, and opening some child folder (/home/test/my_julia_project) which contains a project.toml. If cmd_cwd is not set to the folder, or child folder, of the directory containing project.toml, then the current project resolution in the julia launch script will fail, since it uses the cwd determined by neovim's starting location.

Reply:
Okay, I believe that from the perspective of the julia process running the language server it won't matter (@pfitzseb ?). Is it used for something other than spawning the julia process?

Reply:
Yes, this is only necessary for Base.current_project() to find the correct project.toml, it should not be removed.

Reply:
(It shouldn't matter for Julia, but it will matter for neovim...)

Reply:
it does matter for julia, as finding the project.toml is relative to the cwd of the julia instance

Reply:
Yea. Will add it back, I simply never use that workflow. In any case, that means that the custom config for using a custom install location also need to include it, right?

Reply:
Yes, if you override on_new_config, you would need to also include that snippet.

I think this is a pretty common usecase, as sometimes users will switch between projects with telescope. Without this, you would need to manually changed the cwd of neovim before initiating the telescope project switcher (see also #1106)

Reply:
Yea okay. It just seems a bit annoying that if the default configuration uses `on_new_config` it needs to be duplicated in a user-overridden one.

Reply:
couldn't we have users override cmd instead of overriding on_new_config? I'm not sure why this needs to be done in on_new_config as opposed to appending to cmd ` --project=/path/to/my/julia/environment`